go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/anchore/go-testutils v0.0.0-20200925183923-d5f45b0d3c04
 	github.com/anchore/go-version v1.2.2-0.20200701162849-18adb9c92b9b
 	github.com/anchore/stereoscope v0.0.0-20200925184903-c82da54e98fe
+	github.com/apex/log v1.3.0
```

Comment:
what happened here?

Reply:
Thanks for catching that, I imported log originally to see if I could provide some additional information on a read failure.